src/liberty/dashboard.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022 IBM Corporation.
+ * Copyright (c) 2024 IBM Corporation.
```

Comment:
Why did you update this copyright if there are no changes in this file?

Reply:
Reverted this change